src/helpers/backendCall.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: 'https://caret-bn-backend-staging.herokuapp.com/api/v1',
```

Comment:
Export this url to a utility file and import it. That file will hold the endpoints and make it possible to change endpoints based on the environment.